rero_ils/dojson/utils.py

```diff
@@ -528,7 +528,8 @@ def extract_description_from_marc_field(self, key, value, data):
                 re.IGNORECASE)
             match = regexp.search(extent)
             if match:
-                add_data_and_sort_list('duration', [match.group(1)], data)
+                duration = match.group(1).strip('()')
```

Comment:
strip('()') will strip all () from the beginning end the end. `(((()))()())1234(((())))(())()` -> `1234`
Is this ok for us?

Reply:
@reropag?

Reply:
ok